blog-service/2023-06-02-alerts.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Muting Schedules (Alerts)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - monitors
+  - alerts
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+New - we're happy to announce the release of **Muting Schedules**, a new feature that allows you to pause alert monitor notifications according to a schedule that you define. You can apply this setting to individual alerts and/or folders. For folders, this will mute all nested monitors and subfolders.
+
+Some scenarios where you might want to mute your alerts include your org's scheduled system maintenance and off-business hours activity. [Learn more](/docs/alerts/monitors/alert-response#muting-schedules).
```

Comment:
`alerts` -> `monitor notifications`.
(context: alerts continue to trigger and resolve during these windows, just notifications get muted).